src-backend/api/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls import url, include
 from views import ProcedureViewSet, PageViewSet, ElementViewSet
 from rest_framework import routers
+from signals import page_pre_save
```

Comment:
The reason why I added this here is because we have to import the `page_pre_save` file somewhere, but we can't do it in `models.py` since the `page_pre_save.py` file requires it, which would result in circular dependency. So we load it here in `urls.py`